architecture/core_concepts/builds_and_image_streams.adoc

```diff
@@ -143,35 +143,14 @@ Pipeline workflows are defined in a Jenkinsfile,
 either embedded directly in the build configuration,
 or supplied in a Git repository and referenced by the build configuration.
 
-The first time a project defines a build configuration using a Pipeline
```

Comment:
I created the Pipeline build strategy module for 4.0 and removed the Jenkins autoprovisioning content in https://github.com/openshift/openshift-docs/pull/13000.

As I convert content, I'll continue updating this PR until it is complete in the 4.0 branch.